pom.xml

```diff
@@ -441,31 +441,26 @@ under the License.
           <plugin>
             <groupId>net.nicoulaj.maven.plugins</groupId>
             <artifactId>checksum-maven-plugin</artifactId>
-            <version>1.9</version>
+            <version>1.10</version>
             <executions>
               <execution>
                 <id>source-release-checksum</id>
                 <goals>
-                  <goal>files</goal>
+                  <goal>artifacts</goal>
                 </goals>
+                <configuration>
+                  <algorithms>
+                    <algorithm>SHA-512</algorithm>
+                  </algorithms>
+                  <!-- https://maven.apache.org/apache-resource-bundles/#source-release-assembly-descriptor -->
+                  <includeClassifiers>source-release,source-release-tar</includeClassifiers>
```

Comment:
Where did you get the `source-release-tar` classifier from?

Reply:
My bad, I think all descriptors use the same classifier (https://github.com/apache/maven-resources/blob/e9eaa1baebb72795555eba015eac9b40f288e931/apache-source-release-assembly-descriptor/src/main/assemblies/source-release.xml#L22)